package.json

```diff
@@ -361,13 +361,24 @@
                     ".tpl"
                 ],
                 "configuration": "./language-configuration.json"
+            },
+            {
+                "id": "kubernetes-output",
```

Comment:
Pardon my ignorance, but I don't understand how this identifies _our_ output as subject to the colorisation rules without affecting other output.  I realise this is probably a limitation of my understanding...!

Reply:
The vscode output channel is using mimetype `text/x-code-output`. It seems every extension could extend the highlight rules, and it affects all outputs.